tests/test_feature_price_emission.py

```diff
@@ -0,0 +1,126 @@
+from message_ix import Scenario
```

Comment:
In this file:
- add a docstring for `model_setup()`
- add a plain language comment stating the expected behaviour for each assertion. I'll add an example below.